dcicutils/env_utils.py

```diff
@@ -0,0 +1,21 @@
+FOURFRONT_STG_OR_PRD_TOKENS = ['webprod', 'blue', 'green']
+FOURFRONT_STG_OR_PRD_NAMES = ['staging', 'stagging', 'data']
+CGAP_STG_OR_PRD_TOKENS = []
+CGAP_STG_OR_PRD_NAMES = ['fourfront-cgap', 'fourfront-cgap-green', 'fourfront-cgap-blue']
```

Comment:
Hopefully these can be `cgap-green` and `cgap-blue` but I suspect we will run into the hard coding problem again if that change is made (to be done later).

Reply:
Yeah, I figured when we come up with an official new naming scheme we can fix it here. This gives us a little flex to experiment in the old paradigm.